packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/optimize-ciemss-operation.ts

```diff
@@ -90,14 +90,6 @@ export const OBJECTIVE_FUNCTION_MAP = [
 	{ label: 'upper bound', value: InterventionObjectiveFunctions.upperbound }
 ];
 
-export const blankIntervention: Intervention = {
-	name: 'New Intervention',
-	appliedTo: '',
-	type: InterventionSemanticType.Parameter,
-	staticInterventions: [{ timestep: Number.NaN, value: Number.NaN }],
-	dynamicInterventions: []
-};
-
 export const blankInterventionPolicyGroup: InterventionPolicyGroupForm = {
```

Comment:
Sorry... could you put this one with the `blankIntervention`?

Reply:
I'll actually keep this here since it's used exclusively in this operation and a lot of optimize enums and types are used in it.